fastly/alerts_test.go

```diff
@@ -23,14 +23,25 @@ func TestClient_FastlyAlerts(t *testing.T) {
 		Dimensions:         testDimensions,
 		EvaluationStrategy: testEvaluationStrategy,
 		IntegrationIDs:     []string{},
-		Metric:             ToPointer("status_5xx"),
+		Metric:             ToPointer("status_5xx_rate"),
 		Name:               ToPointer("test name"),
 		ServiceID:          ToPointer(TestDeliveryServiceID),
 		Source:             ToPointer("domains"),
 	}
 
-	// Test
 	var err error
+
+	// Enable Product - Domain Inspector
+	c := DefaultClient()
+	_, err = c.EnableProduct(&ProductEnablementInput{
+		ProductID: ProductDomainInspector,
+		ServiceID: TestDeliveryServiceID,
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
We either enable `Domain Inspector` or we remove `TestClient_FastlyAlerts` entirely.